### PR TITLE
solution and test for #16

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: solutions/16.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  for (let i = 0; i < array.length; i++) {
+    if (map[array[i]]) {
+      return true;
+    }
+    map[array[i]] = true;
+  };
+  return false;
+}
+
+array = [1,2,3];
```
Comment (songz) [inline solutions/16.js]: delete

File: test/16.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const expect = require('chai').expect;
+const duplicates = require('../solutions/16');
+const hasDuplicates = duplicates.hasDuplicates;
+
+describe('check duplicates', () => {
+  it('should return true since array has duplicates', () => {
+    const array = [1,2,3,3,4];
+    const expected = true;
+    expect(hasDuplicates(array)).eql(expected);
```
Comment (songz) [inline test/16.js]: y do you need an expected variable? Just use it directly?

Comment (songz) [inline test/16.js]: chai should also have a .to.be.truthy or something like that method that checks for truthiness

File: solutions/16.js (hunk 1 above)
Comment (msach22) [inline solutions/16.js]: fixed in new MR

File: test/16.js (hunk 2 above)
Comment (msach22) [inline test/16.js]: fixed in new mr

**Pull request closed without merging**